Dockerfile

```diff
@@ -27,9 +27,9 @@ RUN service php7.1-fpm stop && \
     service nginx stop && \
     service supervisor stop
 
-ADD conf/supervisord.conf /etc/supervisord.conf
-ADD conf/nginx/sites-enabled/default /etc/nginx/sites-enabled/default
-ADD conf/www.conf /etc/php/7.1/fpm/pool.d/www.conf
+COPY conf/supervisord.conf /etc/supervisord.conf
+COPY conf/nginx/sites-enabled/default /etc/nginx/sites-enabled/default
```

Comment:
Not the scope of this PR. But we should stop writing directly to `sites-enabled`.